tests/topotests/static_routing_with_ebgp/test_static_routes_topo2_ebgp.py

```diff
@@ -0,0 +1,2338 @@
+            assert result is True, "Testcase {} : Failed \n Error: {}".format(
+                tc_name, result
+            )
+
+            nh = NEXT_HOP_IP["nh" + str(nhp)][addr_type]
+            result = verify_rib(
+                tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+            )
+            assert result is True, "Testcase {} : Failed\nError: Routes are"
+            " missing in RIB".format(tc_name)
+
+    dut = "r2"
+    protocol = "static"
+    step("Shut nexthop interfaces N1 to N3 one by one")
+    for intfr in range(0, 3):
+        intf = topo["routers"]["r2"]["links"]["r1-link{}".format(intfr)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, False)
+        for addr_type in ADDR_TYPES:
+            nhp = intfr + 1
+            input_dict_4 = {
+                "r2": {
+                    "static_routes": [
+                        {
+                            "network": PREFIX1[addr_type],
+                            "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                        }
+                    ]
+                }
+            }
+        nh = NEXT_HOP_IP["nh" + str(nhp)][addr_type]
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_4,
+            next_hop=nh,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {}: Failed\nError: Routes are"
+        " still present in RIB".format(tc_name)
+
+    step("No shut the nexthop interfaces N1 to N3 one by one .")
+    for intfr in range(0, 3):
+        intf = topo["routers"]["r2"]["links"]["r1-link{}".format(intfr)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, True)
+        for addr_type in ADDR_TYPES:
+            nhp = intfr + 1
+            input_dict_4 = {
+                "r2": {
+                    "static_routes": [
+                        {
+                            "network": PREFIX1[addr_type],
+                            "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                        }
+                    ]
+                }
+            }
+        nh = NEXT_HOP_IP["nh" + str(nhp)][addr_type]
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes are"
+        " missing in RIB".format(tc_name)
+
+    step("Random shut of the nexthop interfaces")
+    randnum = random.randint(0, 7)
+    # Shutdown interface
+    dut = "r2"
+    step(
+        " interface which is about to be shut no shut between r1 and r2 is " "%s",
+        topo["routers"]["r2"]["links"]["r1-link{}".format(randnum)]["interface"],
+    )
+    intf = topo["routers"]["r2"]["links"]["r1-link{}".format(randnum)]["interface"]
+    shutdown_bringup_interface(tgen, dut, intf, False)
+
+    sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.